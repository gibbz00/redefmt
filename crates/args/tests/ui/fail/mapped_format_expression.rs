fn main() {
    // unused named variable 'y'
    redefmt_args::mapped_format_expression!("{}", 1, y = true);
}
