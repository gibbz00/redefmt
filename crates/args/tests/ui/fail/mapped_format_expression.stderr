error: provided named argument y not used in format string
 --> tests/ui/fail/mapped_format_expression.rs:3:5
  |
3 |     redefmt_args::mapped_format_expression!("{}", 1, y = true);
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `redefmt_args::mapped_format_expression` (in Nightly builds, run with -Z macro-backtrace for more info)
